src/server/handler/handler_rest/info.rs

```diff
@@ -598,7 +598,50 @@ pub async fn pool_blocks_current_epoch(
     Ok(rweb::Json::from(json!(pool_blocks_current_epoch)))
 }
 
-// #[get("pool/live_delegators/{pool_hash}")] // TODO: FixMe (start with dbsync/api.rs)
+#[get("/pool/pool_reward_recipients/{pool_hash}")]
+#[openapi(
+    id = "api.info.pool.pool_reward_recipients",
+    tags("Pool"),
+    summary = "The quantity of delegators that received rewards last time (epoch) the given pool was a slot leader."
+)]
+pub async fn pool_reward_recipients(
+    pool_hash: String,
+    #[filter = "with_auth"] _user_id: String,
+) -> Result<Json<serde_json::Value>, Rejection> {
+    let dp = crate::DataProvider::new(crate::DBSyncProvider::new(crate::Config {
+        db_path: std::env::var("DBSYNC_URL").unwrap(),
+    }));
+
+    let pool_reward_recipients = dp
+        .pool_reward_recipients(&pool_hash)
+        .await
+        .map_err(|_| RESTError::Custom("Couldn't find the quantity of delegators that received rewards".to_string()))?;
+
+    Ok(rweb::Json::from(json!(pool_reward_recipients)))
+}
+
+#[get("/pool/last_reward_earned_epoch/{pool_hash}")]
+#[openapi(
+    id = "api.info.pool.last_reward_earned_epoch",
+    tags("Pool"),
+    summary = "The last epoch when the given pool gave rewards to delegators"
+)]
+pub async fn pool_last_reward_earned_epoch(
+    pool_hash: String,
+    #[filter = "with_auth"] _user_id: String,
+) -> Result<Json<serde_json::Value>, Rejection> {
+    let dp = crate::DataProvider::new(crate::DBSyncProvider::new(crate::Config {
+        db_path: std::env::var("DBSYNC_URL").unwrap(),
```

Comment:
We should avoid using `unwrap` like this, start having a better error handling for new API will reduce the amount of refactoring that we will need to do in the future.

Reply:
there's pros and cons. Unwrap tells you exactly which line the error comes from, and the question mark operator doesn't tell you that.

Reply:
Unwraping can create runtime panic unless the operation is not fallible, which isn't the case here. It also increase cognitive load, and is  not the best way to deal with errors.

Is it the intention to panic in an HTTP handler, we should do that because that will create internal server error which is avoidable by checking that the environment variable is set at startup.

Reply:
I think we need to wait until Torben comes back before we do that. As I understand it, he wants it to be like this for now. You can talk with him about it when he comes back. Until then I think we can just follow the pre-existing pattern